crest/Assets/Crest/Crest/Scripts/LodData/Settings/SimSettingsAnimatedWaves.cs

```diff
@@ -23,6 +23,13 @@ public partial class SimSettingsAnimatedWaves : SimSettingsBase
 
         public const string k_HelpURL = Internal.Constants.HELP_URL_BASE_USER + "ocean-simulation.html" + Internal.Constants.HELP_URL_RP + "#animated-waves-settings";
 
+        [Tooltip("PREVIEW: Set this to 2 to improve wave quality. In some cases like flowing rivers this can make a substantial difference to visual stability."
+            + "\n\nWe recommend doubling the Lod Data Resolution on the OceanRenderer component to preserve detail after making this change, but note that this will "
+            + "consume 4x more video memory until we are able to optimise data usage further, so apply this change with caution.")]
+        [SerializeField, Range(1f, 4f)]
+        float _waveResolutionMultiplier = 1f;
+        public float WaveResolutionMultiplier => _waveResolutionMultiplier;
```

Comment:
Should the divide-by-zero guard be applied here instead? There are a couple of lines missed below.

Reply:
D'oh - i literally had the same idea but thought "hm no its only 2 places so its not worth it". Thanks did that now

Reply:
Looks good